src/Umbraco.Web.UI.Client/src/assets/lang/da-dk.ts

```diff
@@ -536,7 +536,7 @@ export default {
 		linkToMedia: 'Link til medie',
 		selectContentStartNode: 'Vælg startnode for indhold',
 		selectMedia: 'Vælg medie',
-		chooseMedia: 'Vælg medie startnode',
+		chooseMediaStartNode: 'Vælg medie startnode',
```

Comment:
Should it be `selectMediaStartNode` to keep to consistent with content?

Reply:
@bjarnef That is a good point, I think it makes total sense :-)